airflow-core/src/airflow/utils/helpers.py

```diff
@@ -204,7 +204,7 @@ def build_airflow_dagrun_url(dag_id: str, run_id: str) -> str:
     http://localhost:8080/dags/hi/runs/manual__2025-02-23T18:27:39.051358+00:00_RZa1at4Q
     """
     baseurl = conf.get("api", "base_url", fallback="/")
-    return urljoin(baseurl, f"dags/{dag_id}/runs/{run_id}")
+    return urljoin(f"{baseurl.rstrip('/')}" + "/", f"dags/{dag_id}/runs/{run_id}")
```

Comment:
f string seems useless.
```suggestion
    return urljoin(baseurl.rstrip('/') + "/", f"dags/{dag_id}/runs/{run_id}")
```

Reply:
Actually `urljoin` will take care of stripping extra `/`, as ash mentionned this is similar to `urljoin(baseurl + "/", "dags/{dag_id}/runs/{run_id}")`. (If there is no '/' we add it and join, if there is already a '/' we add another one, and urljoin will take care of resolving the path)